.github/workflows/validate-release.yml

```diff
@@ -1,21 +1,33 @@
-name: Validate Apache Release
-
+name: "Validate Apache Release"
+#latest please refer: https://github.com/apache/incubator-hugegraph-doc/blob/master/.github/workflows/validate-release.yml
```

Comment:
can we use the remote script and delete this file?

Reply:
> can we use the remote script and delete this file?

ok, we can see the apache release validate ci in hugegraph-doc